source/common/http/hash_policy.cc

```diff
@@ -39,15 +42,34 @@ class HeaderHashMethod : public HashMethodImplBase {
                                     const StreamInfo::FilterStateSharedPtr) const override {
     absl::optional<uint64_t> hash;
 
-    // TODO(mattklein123): Potentially hash on all headers.
     const auto header = headers.get(header_name_);
     if (!header.empty()) {
+      absl::InlinedVector<std::string, 1> rewritten_header_values;
```

Comment:
nit: move this closer to where it's used, e.g. define it on L59

Reply:
done